### ISSUE TITLE
Alpha hex support

### ISSUE THREAD

**Issue by Soviut** [less/less.js#1013]
#697 proposes single and double character hex color support (eg: `#E` would convert to `#EEEEEE`).

In the spirit of this, I'm proposing support for alpha hex support.

`#FF000080`

The extra two hex digits at the end would represent the alpha/opacity. LESS could convert this to:

`rgba(255, 0, 0, 0.5)`

Producing a pure red with 50% opacity.

**Comment by ashsearle** [comment]
Can this be reopened bearing in mind 4 and 8-digit hex are defined in [CSS Color Module Level 4
](https://www.w3.org/TR/css-color-4/#hex-notation) and already [supported in various browsers](https://caniuse.com/#feat=css-rrggbbaa)?

**Comment by seven-phases-max** [comment]
@ashsearle The feature already implies automatically as a part of #2986 (though I'll add a remark there just in case).

**Comment by matthew-dean** [comment]
I assume Less could auto-convert `#RRGGBBAA` to `rgba()`?

**Comment by seven-phases-max** [comment]
Yes, it's just a matter of adding add a few lines to handle `#4` and `#8` cases everywhere (it's not too many) `#3` and `#6` things are parsed.

**Comment by LordPachelbel** [comment]
I don't know when it happened, but at some point `rgba()` became able to convert `#rrggbbaa` colors to `rgba()`.

Example:
`box-shadow: 0 3px 6px rgba(#00000029);` will output as `box-shadow: 0 3px 6px rgba(0, 0, 0, 0.16078431);`

**It isn't documented** but it should be!

**Comment by Krinkle** [comment]
Support for `#RRGGBBAA` was released in 2018 with Less.js 3.8.1, via https://github.com/less/less.js/issues/3290 and https://github.com/less/less.js/pull/3291.
